includes/admin/tools/export/give-export-donations-functions.php

```diff
@@ -189,6 +189,10 @@ function give_export_donation_form_search_args( $args ) {
 	$fields = isset( $_POST['fields'] ) ? $_POST['fields'] : null;
 	parse_str( $fields );
 
+	if ( ! empty( $give_forms_categories ) || ! empty( $give_forms_tags ) ) {
```

Comment:
@raftaar1191 if ajax enabled for form for category and tag then why we have `-1` here

Reply:
@raftaar1191 everything is good here